### PR TITLE
Update to support additional adding of apps

### PR DESCRIPTION
**Pull request by rvsia**
Needed for https://github.com/RedHatInsights/sources-ui/pull/168

This PR updates exported components to support adding of applications to sources in UI

- Imports additional functions as helpers
- Allows to disable password requirements (in UI, we cannot pre-filled them as these values are not obtainable by API)
- Handling undefined/null values in Summary screen

TODO
- [x] disable password requirements

### CODE REVIEW

File: packages/sources/src/addSourceWizard/schemaBuilder.js [hunk 1]
```diff
@@ -25,13 +25,24 @@ export const getAdditionalStepFields = (fields, stepKey) => fields.filter(field
 
 export const getNoStepsFields = (fields, additionalStepKeys = []) => fields.filter(field => !field.stepKey || additionalStepKeys.includes(field.stepKey));
 
-export const injectAuthFieldsInfo = (fields, type, auth, applicationName) => fields.map((field) => {
+export const removeRequiredValidator = (validate = []) =>
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/schemaBuilder.js]: Why is this here? Is it possible to use the form-level validation and have the login there instead of doing this.

Comment (rvsia) [reply packages/sources/src/addSourceWizard/schemaBuilder.js]: Not sure how. I need to disable required validators on password values when editing (aka attaching application to a source which already has the right credentials) because the password is not obtainable by the API.

Comment (Hyperkid123) [reply packages/sources/src/addSourceWizard/schemaBuilder.js]: Can you make the field conditional?

Comment (rvsia) [reply packages/sources/src/addSourceWizard/schemaBuilder.js]: But user should be able to change if he wants to do it. (HelperText is also added with a description.)

It's the same flow as the adding, just with pre-filled values.

**Comment by Hyperkid123** [comment]
Theoretically, you could prefill it with `null` but then you would have to check the value before posting it.
